CodeGen/Generators/NanoFrameworkGenerator.cs

```diff
@@ -140,14 +140,26 @@ public static void Generate(string rootDir, Quantity[] quantities, bool updateNa
                     logger,
                     cancellationToken).Result;
 
-                // NuGet package Version
-                // including preview
-                var mscorlibPackage = packageVersions.Where(v => v.IsPrerelease).OrderByDescending(v => v).First();
-                // stable only
-                //var mscorlibPackage = packageVersions.OrderByDescending(v => v).First();
+                // get NuGet package Version for mscorlib
+                var latestStableVersion = packageVersions.OrderByDescending(v => v).First();
+                var latestPreviewVersion = packageVersions.Where(v => v.IsPrerelease).OrderByDescending(v => v).First();
```

Comment:
It can, of course.
The thing is that I thought to have processing both preview and stable in case we need to specifically target one or the other.
Of course that would require an extra parameter to pass that and adapt all call upwards.

If you don't like it, I can simplify and just grab the latest, regardless of the preview/stable state.

Let me know how you prefer, 😄